dev-environment/docker-compose-h2o.yaml

```diff
@@ -0,0 +1,24 @@
+version: '3.8'
+
+services:
+
+  h2o-server:
+    image: docker.pkg.github.com/prinz-nussknacker/prinz/h2o-server:0.0.1
+    environment:
+      - H2O_WORK_DIR=${H2O_WORK_DIR}
+      - H2O_SERVER_PORT=${H2O_SERVER_PORT}
+    ports:
+      - "${H2O_SERVER_PORT}:${H2O_SERVER_PORT}"
+      - "${H2O_WORK_DIR}:${H2O_WORK_DIR}"
+    volumes:
+      - ./h2o/models:${H2O_WORK_DIR}/models
+      - ./h2o/exports:${H2O_WORK_DIR}/exports
+    networks:
+      - h2o-net
+
+networks:
+  h2o-net:
```

Comment:
do we need this network? it has only one container

Reply:
will eb used when we introduce proxy forr nussknacker use (so I would leave it now)